cpp/examples/utilities/io_source.hpp

```diff
@@ -0,0 +1,184 @@
+/*
```

Comment:
Nothing new here, all moved code from `io_source.hpp/cpp` earlier residing in `parquet_io`